docs/user-documentation/soc-and-feeds.md

```diff
@@ -0,0 +1,126 @@
+
+Publishers are the single owners of feed chunks and are the only ones able to post updates to their feed. Posting an update requires (1) constructing the identifier from the topic and the correct index and (2) signing it concatenated together with the hash of the arbitrary content of the update.
+
+Conversely, users can consume a feed by retrieving the chunk by its address. Retrieving an update requires the consumer to construct the address from the owner’s public key and the identifier. To calculate the identifier they need the topic and the appropriate index. For this they need to know the indexing scheme.
+
+Feeds enable Swarm users to represent a sequence of content updates. The content of the update is the payload that the feed owner signs against the identifier. The payload can be a swarm reference from which the user can retrieve the associated data.
+
+### Topic
+
+In Swarm `topic` is a 32-byte arbitrary byte array. It's possible to choose an arbitrary topic for each application and then knowing someone's (or something's) address it's possible to find their feeds. Also this can be the hash of one or more human readable strings, specifying the topic and optionally the subtopic of the feed. There is a helper function provided for that:
+
+```js
+const topic = bee.makeFeedTopic('my-dapp.eth/outbox')
+```
+
+### Reading feeds
+
+In order to read data from a feed we need to make a reader object for a certain `type`, `topic` and `owner`, then we can download the latest update containing a reference.
+
+```js
+const topic = '0000000000000000000000000000000000000000000000000000000000000000'
+const owner = '0x8d3766440f0d7b949a5e32995d09619a7f86e632'
+const feedReader = bee.makeFeedReader('sequence', topic, owner)
+const feedUpdate = await feedReader.download()
+console.log(feedUpdate.reference) // prints the latest reference stored in the feed
+```
+
+### Writing feeds
+
+When writing a feed, typically an immutable content is uploaded first and then its reference is updated in the feed. The `signer` here is the same as with [writing the SOCs](#writing-socs) (with the same caveats!).
+
+```js
+const data = new Uint8Array([1, 2, 3])
+const reference = await bee.uploadData(data)
+const topic = '0000000000000000000000000000000000000000000000000000000000000000'
+const signer = '0x634fb5a872396d9693e5c9f9d7233cfa93f395c093371017ff44aa9ae6564cdd'
+const feedWriter = bee.makeFeedWriter('sequence', topic, signer)
+const response = await feedWriter.upload(reference)
+```
+
+### Using feed manifest
+
+One of the most common use cases for feeds is to store mutable data in an immutable address. For example, when hosting a website to Swarm, we may want its address stored in ENS but we don't want to pay for changing the reference every time the site is updated.
```

Comment:
This page is IMHO really good for a clueless user (like I am) and I would love to see it even repeated right under the `## Feeds` headline. It would be great to learn first why I may want to use Feeds and SOC for that matter.

Reply:
Added a copy of this paragraph to the `## Feeds` section.